sdk/google/drive/file creator/a944f36b-bdfe-4e46-bfc2-a60548bbef56.json

```diff
@@ -0,0 +1,156 @@
+{
+  "name": "sdk/google/Drive/File creator",
+  "interface": {
+    "inputs": {
+      "80b2ea3c-ab45-4e08-9a62-83f87b7c5c87": {
+        "type": "string",
+        "name": "session Id",
+        "description": "Receives the session id of the drive action.\n\nExample: \n\"drive_session\""
+      },
+      "67d952ea-512f-4028-8a2f-8f768bfa2e7a": {
+        "type": "{string:any}",
+        "name": "file meta data",
+        "description": "Receives the metadata of the file creation.\n\nExample in case of shared drive:\n\n{\n  \"name\": \"Test from CRANQ dynamic\",\n  \"driveId\": \"0AFITUOLHhs3TUk9PVA\",\n  \"corpora\": \"drive\",\n  \"parents\": [\n    \"FOLDER_ID\"\n  ],\n  \"mimeType\": \"application/vnd.google-apps.spreadsheet\"\n}\n\n\nExample in case of shared folder:\n\n{\n  \"name\": \"Test from CRANQ dynamic\",\n  \"parents\": [\n    \"FOLDER_ID\"\n  ]\n  \"mimeType\": \"application/vnd.google-apps.spreadsheet\"\n}\n"
+      },
+      "2890771f-bac0-4da3-998d-8f9530fcc98d": {
+        "type": "boolean",
+        "name": "supports all drives",
+        "description": "Receives `supportsAllDrives` parameter whether create should support both drives and shared drives.\n\nExample: \n\"true\""
```

Comment:
Still unclear what is the `supportsAllDrives` we refer here. Is it a google SDK parameter?

Reply:
Yes. I don't want to put here all the google sdk documentations. The one who ones to use this basic node needs to be aware of the usage of the sdk